netmanager/src/redux/Join/actions.js

```diff
@@ -351,7 +342,7 @@ export const registrationSuccess = data => {
 /************************* Login a new User  *********************************/
 export const loginUser = userData => dispatch => {
   console.log('the login URL ' + constants.LOGIN_USER_URI);
-  const tenant = 'kcca'
+  const tenant = userData.organization || "kcca"
```

Comment:
Does this mean that if the organisation does not exist, use "kcca"?

On a related note, just entered some random org name and was unable to proceed, could we add some alert to inform user about the issue ("your organisation does not exist on the platform").
On that note, I have made a new commit to the [corresponding backend PR](https://github.com/airqo-platform/AirQo-api/pull/121) (please PULL) to send an appropriate error response (see below) which we can use accordingly.
```
{
    "success": false,
    "message": "organization does not exist"
}
```

Reply:
I do not have clear recommendations about how we can handle this scenario but feel free to explore and choose best approach. :)

Reply:
> I do not have clear recommendations about how we can handle this scenario but feel free to explore and choose best approach. :)

Noted. Thank you. I will fix this.

Reply:
> Does this mean that if the organisation does not exist, use "kcca"?
> 
> On a related note, just entered some random org name and was unable to proceed, could we add some alert to inform user about the issue ("your organisation does not exist on the platform").
> On that note, I have made a new commit to the [corresponding backend PR](https://github.com/airqo-platform/AirQo-api/pull/121) (please PULL) to send an appropriate error response (see below) which we can use accordingly.
> 
> ```
> {
>     "success": false,
>     "message": "organization does not exist"
> }
> ```

I will have to remove the or part. You cannot reach this point without providing an org. will fix and work on displaying any errors back to the user

Reply:
> Does this mean that if the organisation does not exist, use "kcca"?
> 
> On a related note, just entered some random org name and was unable to proceed, could we add some alert to inform user about the issue ("your organisation does not exist on the platform").
> On that note, I have made a new commit to the [corresponding backend PR](https://github.com/airqo-platform/AirQo-api/pull/121) (please PULL) to send an appropriate error response (see below) which we can use accordingly.
> 
> ```
> {
>     "success": false,
>     "message": "organization does not exist"
> }
> ```

@Baalmart This has been addressed